Makefile

```diff
@@ -35,6 +35,13 @@ milvus: build-cpp print-build-info
 		GO111MODULE=on $(GO) build -ldflags="-r $${RPATH} -X '$(OBJPREFIX).BuildTags=$(BUILD_TAGS)' -X '$(OBJPREFIX).BuildTime=$(BUILD_TIME)' -X '$(OBJPREFIX).GitCommit=$(GIT_COMMIT)' -X '$(OBJPREFIX).GoVersion=$(GO_VERSION)'" \
 		${AARCH64_FLAG} -o $(INSTALL_PATH)/milvus $(PWD)/cmd/main.go 1>/dev/null
 
+milvus-gpu: build-cpp-gpu print-build-info
```

Comment:
Why we need this. Some similar Flag like `WITH_DISKANN` doesn't have this.

Reply:
because we need release 2 images, "milvus" for cpu only, "milvus-gpu" for cpu & gpu